1.1/spec/11-Part-08.adoc

```diff
@@ -281,11 +281,22 @@ geo:hasSerialization
 NOTE: this property is the generic property used to connect a Geometry with its serialization. GeoSPARQL also contains a number of sub properties of this one for connecting serializations of common types with geometries, for example <<Property: geo:asGeoJSON, as GeoJSON>> which can be used for GeoJSON <<GEOJSON>> literals.
 
 === Geometry Serializations
-This section establishes the requirements for representing Geometry data in RDF based on different systems.
+This section establishes the requirements for representing Geometry data in RDF literals, according to different non-RDF systems.
+
+GeoSPARQL presents specializations of the `geo:hasSerilization` property for indicating particular serializations and specialized datatype literals for containing them but does not provide comprehensive definitions of their content since these are given in standards external to GeoSPARQL, all of which are referenced.
+
+GeoSPARWL does present some requirements for literal structure which extend the serialization-defining standards, for example the requirement to allow indications of spatial reference systems within WKT geometry representations.
```

Comment:
Typo: GeoSPARWL -> GeoSPARQL

Reply:
fixed in https://github.com/opengeospatial/ogc-geosparql/pull/288/commits/209d8b45eab5c52bb12f8ed1675f0303afc13a31